The fix for https://bugreports.qt.io/browse/QTBUG-76244 was not complete.
Further quoting is needed to deal with empty variables.
-- Eric Hameleers <alien@slackware.com>

--- qt-everywhere-src-5.13.0/qtbase/mkspecs/features/data/cmake/Qt5BasicConfig.cmake.in.orig	2019-06-13 06:18:18.000000000 +0200
+++ qt-everywhere-src-5.13.0/qtbase/mkspecs/features/data/cmake/Qt5BasicConfig.cmake.in	2019-07-09 19:37:15.253291684 +0200
@@ -57,8 +57,8 @@
 
     if(EXISTS \"${prl_file_location}\")
         file(STRINGS \"${prl_file_location}\" _prl_strings REGEX \"QMAKE_PRL_LIBS[ \\t]*=\")
-        string(REGEX REPLACE \"QMAKE_PRL_LIBS[ \\t]*=[ \\t]*([^\\n]*)\" \"\\\\1\" _static_depends ${_prl_strings})
-        string(REGEX REPLACE \"[ \\t]+\" \";\" _static_depends ${_static_depends})
+        string(REGEX REPLACE \"QMAKE_PRL_LIBS[ \\t]*=[ \\t]*([^\\n]*)\" \"\\\\1\" _static_depends \"${_prl_strings}\")
+        string(REGEX REPLACE \"[ \\t]+\" \";\" _static_depends \"${_static_depends}\")
         string(REGEX REPLACE \"[ \\t]+\" \";\" _standard_libraries \"${CMAKE_CXX_STANDARD_LIBRARIES}\")
         set(_search_paths)
         string(REPLACE \"\\$\\$[QT_INSTALL_LIBS]\" \"${_qt5_install_libs}\" _static_depends \"${_static_depends}\")
